src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -18,85 +28,506 @@ def _get_test_data_file(filename):
     return os.path.join(curr_dir, 'data', filename).replace('\\', '\\\\')
 
 
+def install_helm_client():
+    # Return helm client path set by user
+    if os.getenv('HELM_CLIENT_PATH'):
+        return os.getenv('HELM_CLIENT_PATH')
```

Comment:
is this needed in test env? no one's going to set the env variables right?

Reply:
Right , resolved